pkg/genericcontrolplane/options/options.go

```diff
@@ -115,9 +113,6 @@ func NewServerRunOptions() *ServerRunOptions {
 	// disable the watch cache
 	s.Etcd.EnableWatchCache = false
 
-	// TODO: turn off the admission webhooks for now
-	s.Admission.DefaultOffPlugins.Insert(validating.PluginName, mutating.PluginName)
```

Comment:
@shawn-hurley I remove you insert few lines below. Is that correct? Any reason it should be off by default?

Reply:
They should be off by default for us because they are not logical cluster-aware. For the generic control plane, I think it could be fine, but we will need to double-check in the KCP pr that this is off.